test_runner/fixtures/utils.py

```diff
@@ -148,7 +148,7 @@ def get_scale_for_db(size_mb: int) -> int:
 
 
 ATTACHMENT_NAME_REGEX: re.Pattern = re.compile(  # type: ignore[type-arg]
-    r"flamegraph\.svg|regression\.diffs|.+\.(?:log|stderr|stdout|filediff|metrics|html)"
```

Comment:
I think we'd better leave svg, since at least our layer visualizer tool produces them.

Reply:
Hmm, are you proposing to include `.*\.svg` here? That would catch *more* files than what we had before this PR. It does sound reasonable to me, although we don't produce any other .svg files in the CI currently AFAIK.